sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ProcessorUtils.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+
+namespace Azure.Messaging.ServiceBus
+{
+    internal static class ProcessorUtils
+    {
+        /// <summary>
+        ///
+        /// </summary>
+        /// <param name="lockedUntil"></param>
+        /// <returns></returns>
+        internal static TimeSpan CalculateRenewDelay(DateTimeOffset lockedUntil)
```

Comment:
Do we handle delay in multiple places now? Why?

Reply:
Message lock renewal is used for non-session processor.